refactor/results/requirements.txt

```diff
@@ -0,0 +1,66 @@
+alembic==1.7.6
```

Comment:
can this list be trimmed down at all?  we should at least separate out dev requirements (which right now go in tests/requirements.txt) from core user reqs